src/guide/1/pubsub/javascript/snippets/pubsubjs.js

```diff
@@ -0,0 +1,34 @@
+const {
+  Kuzzle,
+  WebSocket
+} = require('kuzzle-sdk');
+const kuzzle = new Kuzzle(new WebSocket('kuzzle'));
+// Creates a filter that defines that the 'message' field exists
+const filter = {exists: {field: 'message'}};
+// Will be triggered each time a document matches the filter
+const callback = (notification) => {
```

Comment:
(optional / nitpicking)

```suggestion
const callback = notification => {
```